### DISCUSSION TITLE
INSTALLS CONSULTATION

### DISCUSSION THREAD

**Discussion by 0xJonaseb11** [Cyfrin/foundry-full-course-cu#305] [Q&A] [answered]

**Comment by shikhar229169** [comment, marked as answer]
You can go to the link: https://github.com/Cyfrin/foundry-full-course-f23

Go to the table of contents and go to the lesson for which you need to install the dependencies and click on code, you will be then redirected to the repo of the lesson, there you can find the forge install links required to install the dependencies.

**Reply by cromewar** [reply]
Thanks @shikhar229169
